lib/embulk/output/mailchimp.rb

```diff
@@ -110,6 +112,19 @@ def add_subscriber(row)
           m[key] = row[col_name] if row[col_name]
         end
 
+        if @grouping_columns && !@grouping_columns.empty?
+          merge_vars[:groupings] = []
+
+          @grouping_columns.each do |group|
+            next unless row[group]
+
+            merge_vars[:groupings] << {
+              'name'   => group,
+              'groups' => row[group].split(','),
```

Comment:
We don't needed empty string rejection?
If `row[group]` is `""`, groups to be an empty array.
(if it is acceptable or necessary behavior, ignore this comment)